### DISCUSSION TITLE
Callout within Columns

### DISCUSSION THREAD

**Discussion by kennygokh** [efemkay/obsidian-modular-css-layout#31] [Q&A]
Hi, is it possible to have multiple callouts within a column? I am trying to design the note to display 2 columns. See screenshot for illustration.
Column 1 - contain multiple callouts
Column 2 - contain a Dataview code

![8BEA1653-1E55-4F0F-99D8-77F0AA31D71A](https://user-images.githubusercontent.com/97268671/226639398-ac7d9094-7a2b-4a13-881a-06e053613e82.jpeg)

**Comment by efemkay** [comment]
as per my response in discord, this should do it. u just have to think properly how to nest it. my advise, do the columns first without applying the multi column layout, once you have the columns ready, then encapsulate those inside main callout multi-column

```
> [!multi-column] Title
>
>> [!note] Column 1
>> 
>>> [!Help]
>>> 
>>
>>> [!Warning]
>>> 
>
>> [!info] Column 2 - Dataview
>> 
>> ```dataview
>> LIST
>> FROM "sample data A"
>> ```
```

![image](https://user-images.githubusercontent.com/42369515/227701921-436f1aa6-9736-4e0f-9acd-4e1b48acd231.png)
